chapters/resources.txt

```diff
@@ -526,10 +526,11 @@ include::{generated}/api/structs/VkBufferViewCreateInfo.txt[]
 ****
   * [[VUID-VkBufferViewCreateInfo-offset-00925]]
     pname:offset must: be less than the size of pname:buffer
-  * [[VUID-VkBufferViewCreateInfo-offset-02749]]
-    If the <<features-texelBufferAlignment,texelBufferAlignment>> feature is
-    not enabled, pname:offset must: be a multiple of
+ifndef::VK_EXT_texel_buffer_alignment[]
+  * [[VUID-VkBufferViewCreateInfo-offset-00926]]
```

Comment:
This is reverting to old VUID and old VU text for the non-extension version of the VU.